ajax/libs/jQuery.BgSwitcher/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jQuery.BgSwitcher",
+  "filename": "jquery.bgswitcher.min.js",
+  "version": "0.4.3",
+  "description": "Switch the background image with using effect.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rewish/jquery-bgswitcher.git"
+  },
+  "author": "rewish",
```

Comment:
@extend1994 We can use `author` info as the same as upstream. ref: https://github.com/rewish/jquery-bgswitcher/blob/master/bgswitcher.jquery.json#L5-L9.